pkg/artifactory/resource/repository/local/resource_artifactory_local_debian_repository.go

```diff
@@ -33,7 +33,7 @@ var DebianLocalSchema = util.MergeMaps(
 
 type DebianLocalRepositoryParams struct {
 	RepositoryBaseParams
-	TrivialLayout           bool     `hcl:"trivial_layout" json:"debianTrivialLayout,omitempty"`
+	TrivialLayout           bool     `hcl:"trivial_layout" json:"debianTrivialLayout"`
```

Comment:
DOes this attribute have a default value? Otherwise, we can get a null pointer, right?

Reply:
Good catch. Let me check.